src/pages/AdvancedSearch/UnitsLink/UnitsLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { type PropsWithChildren } from "react";
+import { Link } from "react-router-dom";
+
+import { useModelByUUIDDetails } from "components/hooks";
+import urls from "urls";
+
+type Props = {
+  uuid: string;
+  appName: string | null;
+  unitId: string | null;
+} & PropsWithChildren;
+
+const UnitsLink = ({ uuid, appName, unitId, children }: Props): JSX.Element => {
```

Comment:
Both `UnitsLink` and `UnitsMachineLink` look similar, however, there are a few subtle differences between both of them and thus, I didn't know if we should combine both of them in a single component.